code/main_0417.py

```diff
@@ -145,25 +126,22 @@
         weight_tmp = weight_cov_val
         covl_tmp = covl_xy_val
 
-    for batch_id in range(0, num_steps):
-        X_batch = X_batches[batch_id]
-        Y_batch = Y_batches[batch_id]
-
-        ## update posterior mean/covariance
-        try:
-            _, summary = sess.run([train_mean, summary_op],
-                                  feed_dict={X: X_batch,
-                                             Y_true: Y_batch,
-                                             H_inv: weight_cov_val,
-                                             Phi_y: covl_xy_val})
-
-        except:
-            print("\n================================\n"
-                  "Problem occurred at Step {}\n"
-                  "================================".format(batch_id))
```

Comment:
I removed this training loop based on `train_mean` since it seems to impact the covariance estimate